### ISSUE TITLE
fix: 'TypeError: 'NoneType' object is not iterable' after package update 0.5.2 -> 0.6.0

### ISSUE THREAD

**Issue by AIGeneratedUsername** [JWCook/aiohttp-client-cache#110]
### The problem
In my CI tests, I have the error after package version bump 0.5.2 -> 0.6.0

```
src/common/apis/restful_api.py:516: in _request
    async with self.session.request(method=method, url=url, params=params, headers=headers, **kwargs) as response:
.venv/lib/python3.10/site-packages/aiohttp/client.py:1138: in __aenter__
    self._resp = await self._coro
.venv/lib/python3.10/site-packages/forge/_revision.py:322: in inner
    return await callable(*mapped.args, **mapped.kwargs)
.venv/lib/python3.10/site-packages/aiohttp_client_cache/session.py:45: in _request
    self._cookie_jar.update_cookies(response.cookies, response.url)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <aiohttp.cookiejar.CookieJar object at 0x7f13e3462ad0>, cookies = None
response_url = URL('www.test-aiohttp-client-cache-url.com')
    def update_cookies(self, cookies: LooseCookies, response_url: URL = URL()) -> None:
        """Update cookies."""
        hostname = response_url.raw_host
    
        if not self._unsafe and is_ip_address(hostname):
            # Don't accept cookies from IPs
            return
    
        if isinstance(cookies, Mapping):
            cookies = cookies.items()
    
>       for name, cookie in cookies:
E       TypeError: 'NoneType' object is not iterable
.venv/lib/python3.10/site-packages/aiohttp/cookiejar.py:167: TypeError
```

My guess is that either a new package version (0.6) is not compatible with an existing cache from the version 0.5, or a new bug was introduced in the 0.6 version. If the first case, maybe the old cache must be auto-cleaned.

### Environment
- aiohttp-client-cache version: 0.6.0
- Python version: 3.10

--- 

I do not know whether this important or not, but my test in my CI looks like (simplified):
```python
async def test_aiohttp_client_cache(url: str) -> None:

    async def mock_response(*args: Any, **kwargs: Any) -> CachedResponse:
        return CachedResponse("GET", "url", 200, url, None, encoding="utf-8")  # type: ignore[arg-type]

    with patch.object(ClientSession, "_request", side_effect=mock_response) as mock_request:
           
           # do some requests here in FOR loop...
 
           assert len(mock_request.mock_calls) == 1
```

**Comment by AIGeneratedUsername** [comment]
I do not know whether this important or not, but my test looks like:
```python
    async def mock_response(*args: Any, **kwargs: Any) -> CachedResponse:
        return CachedResponse("GET", "url", 200, url, None, encoding="utf-8")  # type: ignore[arg-type]

    with patch.object(ClientSession, "_request", side_effect=mock_response) as mock_request:
           # do request many here...
           assert len(mock_request.mock_calls) == 1
```

**Comment by JWCook** [comment]
Thanks for the bug report. I think your hunch may be correct. The changes to support cached session cookies may not be compatible with cached responses from 0.5.2, which is something I didn't test.

This should be easy enough to fix, though.

**Comment by JWCook** [comment]
Fixed in v0.6.1. Give that a try and let me know if that works for you.
